<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="firstTbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>231</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="secondTbl">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>361</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="secondDblBox">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="secondIntBox">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Second</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>First</string>
    </property>
   </widget>
   <widget class="QPushButton" name="secondPushBtn">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>60</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Push back</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="firstIntBox">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>140</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="firstPushBtn">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>160</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Push Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eraseBtn">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>260</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Erase</string>
    </property>
   </widget>
   <widget class="QPushButton" name="secondInsertBtn">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>90</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="firstInsertBtn">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>190</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="popBackBtn">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>290</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Pop Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearBtn">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>320</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
